apps/api/views.py

```diff
@@ -416,6 +416,18 @@ def consent(request: Request):
             },
             status_codes=[200],
         ),
+        OpenApiExample(
+            name="GenerateBotMessageAndSendUsingANewSession",
+            summary="Generates a bot message and sends it to the user using a new session",
+            value={
+                "identifier": "part1",
+                "experiment": "exp1",
+                "platform": "connect_messaging",
+                "prompt_text": "Tell the user to do something",
+                "start_new_session": "true",
+            },
+            status_codes=[200],
+        ),
```

Comment:
Do you find these useful in the docs? I'm not sure it's worth having too many examples.

Reply:
Personally no, but I generally like examples. We can take it out if you feel it's cluttering it?

Reply:
I think one or two is great but this one doesn't add much value over the basic one. It just shows the one additional field.